addons/im_livechat/static/src/core/common/thread_model_patch.js

```diff
@@ -36,6 +36,13 @@ patch(Thread.prototype, {
                 }
             },
         });
+        this.livechat_active = Record.attr(undefined, {
+            onUpdate() {
+                if (!this.livechat_active) {
+                    this.markAsRead();
```

Comment:
I believe we should pass the `sync` parameter to true (otherwise we might keep the banner locally)